gossipdb.go

```diff
@@ -1,21 +1,23 @@
 package gossipdb
 
 import (
+	"encoding/json"
+	"fmt"
 	"github.com/hashicorp/memberlist"
 )
 
-type KeyParser interface {
-	ToKey(b []byte) string
+type Pair struct {
+	Key   string
+	Value string
```

Comment:
How about allowing `value` to be `interface{}`, honestly i see no constraint yet on this ?

Reply:
Yup, I was Also thinking the same